.changeset/config.json

```
@@ -22,6 +22,7 @@
       "@udecode/slate-plugins-common",
       "@udecode/slate-plugins-core",
```

looks like I forgot to add csv-serializer in here...
```suggestion
      "@udecode/slate-plugins-core",
      "@udecode/slate-plugins-csv-serializer",
```